.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
-  "editor.formatOnPaste": true,
-  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false,
+  "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll": true
+    "source.fixAll": false
```

Comment:
It seems like everyone hates my settings :D

Reply:
Yes!! It does crazy things, each time I save, everywhere double-quotes and semicolons are introduced, which are not allowed by the current eslint config

Reply:
Double quotes only apply for the `template`-part but not for the `script`-part.
Because of the `bind` of Vue it might that you paste a `this`-variable (see the i18n-parts).
And this rule does the formatting according to the eslint.config.

Reply:
doesn't work for me, maybe I'm missing extensions

Reply:
If you go to extensions in vscode, you should see the recommendations I made or have a look on `extensions.json`

Reply:
I have those installed but they don't do what I would expect

Reply:
So let's just removed that damn formatter preference. In the end it's only a dev preference

Reply:
Haha,
okay. Just saying that it was compliance with the Lint rules